src/lib/components/StickySidebar.tsx

```diff
@@ -0,0 +1,102 @@
+import type { BoxProps } from "@chakra-ui/react";
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Text,
+} from "@chakra-ui/react";
+import type { ReactElement } from "react";
+
+import { CustomIcon } from "./icon";
+import { SwitchToTestnet } from "./SwitchToTestnet";
+
+interface StickySidebarProps extends BoxProps {
+  title?: string;
+  description?: ReactElement;
+  testnetSwitch?: boolean;
+  action?: boolean;
+}
+
+export const StickySidebar = ({
+  title = " Why do I need to submit proposal?",
+  description = (
```

Comment:
Move this out and remove ? for title and description. Since this component will be used in Proposal to store code too